### DISCUSSION TITLE
Github Pages

### DISCUSSION THREAD

**Discussion by NolvusMadeIt** [lihbr/lihbr-apex#16] [Q&A] [answered]
Hi Lucy, dumb question, can this be hosted on github?

**Comment by lihbr** [comment, marked as answer]
Hey, more or less, it requires serverless functions from Netlify to fully work but the main app is only static assets once built. I'm not planning to move to GitHub pages anytime soon though.

What are you working on? Code is open source and under MIT for the sake of sharing how I built it 🙂

**Reply by NolvusMadeIt** [reply]
yes :)

**Comment by alex455393** [comment]
Hello
